tsconfig.json

```diff
@@ -9,9 +9,21 @@
     "resolveJsonModule": true,
     "skipLibCheck": true,
     "sourceMap": true,
-    "strict": true,
-    "moduleResolution": "bundler"
-  }
+    "strict": true
+  },
+  "include": [
+    "./svelte-kit/ambient.d.ts",
+    "./svelte-kit/non-ambient.d.ts",
+    "./src-ui/**/*.js",
+    "./src-ui/**/*.ts",
+    "./src-ui/**/*.svelte",
+    "./.storybook/**/*.js",
+    "./.storybook/**/*.ts",
+    "./.storybook/**/*.svelte",
+    "./*.config.js",
+    "./*.config.ts",
+    "./vitest.diff.ts"
+  ]
```

Comment:
needed to extend the `include` list from .svelte-kit/tsconfig.json; and there is no way to do so without overriding the whole thing.